phpunit/class-rest-blocks-controller-test.php

```diff
@@ -101,7 +101,7 @@ public function test_capabilities( $action, $role, $expected_status ) {
 				$request->set_body_params(
 					array(
 						'title'   => 'Test',
-						'content' => '<!-- wp:core/paragraph --><p>Test</p><!-- /wp:core/paragraph -->',
+						'content' => '<!-- wp:paragraph --><p>Test</p><!-- /wp:paragraph -->',
```

Comment:
Are these test changes related to the schema change? I wouldn't expect the block name to change here.

Reply:
No, not related. I just noticed that the test data here was out of date (we stopped prefixing block markup with `core/` a while ago) 🙂

Reply:
Huh, it's odd that the test didn't fail before then...

Reply:
We still support the core prefix to not break older blocks.

Reply:
Ah, maybe we should have a test for both then? 😉